app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteQuickAddPopup.kt

```diff
@@ -42,11 +40,11 @@ class AutocompleteQuickAddPopup(context: Context, url: String) : PopupWindow() {
     }
 
     fun show(anchor: View) {
-        val y = (anchor.height * Y_POS_MULTIPLIER).roundToInt()
-        super.showAtLocation(anchor, Gravity.CENTER_HORIZONTAL or Gravity.TOP, 0, y)
+        super.showAsDropDown(anchor, X_POS_OFFSET, Y_POS_OFFSET)
     }
 
     companion object {
-        const val Y_POS_MULTIPLIER = 1.5
+        private const val X_POS_OFFSET = -10
```

Comment:
\[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.